core/src/main/java/com/hcl/ecomm/core/servlets/CreateOrderServlet.java

```diff
@@ -101,9 +103,13 @@ private JSONObject jsonItemObj(JSONObject shipData, String customerToken) {
 		JSONObject billAddress = new JSONObject();
 		try {
 			if(customerToken != null && !customerToken.isEmpty()) {
+				if(shipData.getString("email").equalsIgnoreCase(NOT_AVAILABLE)){
```

Comment:
@vishalgupta-hcl : you can add an or condition for empty check for shipData.getString("email") as well.